python/fledge/services/core/server.py

```diff
@@ -1316,21 +1304,35 @@ async def cert_login(ca_cert):
                 raise api_exception.AuthenticationIsOptional
             
             auth_method = category_info['authMethod']['value']
-            ca_cert_name = category_info['authCertificateName']['value']
-
-            verification_id = request.query['_vid']
-            if not len(verification_id.strip()): raise api_exception.VerificationFailed             
+            ca_cert_name = category_info['authCertificateName']['value']          
             
             try:
-                fname = _FLEDGE_ROOT + "/data/.{}".format('managementtoken')
-                with open(fname, 'r', encoding = 'utf-8') as f:
-                    t = f.read()
+                auth_header = request.headers.get('Authorization', None)
             except:
                 raise api_exception.VerificationFailed
             else:
-                if verification_id != t:
+                # check bearer token with service registry for given service
```

Comment:
todo: add checks,
```
if auth_header:
    if not "Bearer " in auth_header:
        log bad Authorization header data
        return UNAUTHORIZED
else:
   return UNAUTHORIZED
```

@pintomax  probably this should move to a def _check_auth_header along with next repeated (among other def/s) code.